core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -64,6 +64,19 @@ private[log] class LogCleanerManager(val logDirs: Array[File], val logs: Pool[To
   @volatile private var dirtiestLogCleanableRatio = 0.0
   newGauge("max-dirty-percent", new Gauge[Int] { def value = (100 * dirtiestLogCleanableRatio).toInt })
 
+  /* a gauge for tracking the number of logCleaner runs */
+  @volatile private var numLogCleanRuns = 0
+  newGauge("num-runs", new Gauge[Int] { def value = numLogCleanRuns })
+
+  /* a gauge for tracking the last time of logCleaner run */
+  @volatile private var logCleanLastRun : Long = System.currentTimeMillis
```

Comment:
I think this would be better as `time-since-last-run` - that way somebody monitoring this through JMX can alert simply if it's greater than a few minutes ago, rather than having to also check the system time (or compare clocks across computers which is notably fragile, particular in the face of NTP changes).